src/me/jupath/tsw/UITweaks/Fix4K/Fix4K.as

```diff
@@ -49,21 +49,11 @@ class me.jupath.tsw.UITweaks.Fix4K.Fix4K extends Plugin {
 	private function fixSkillHive() : Void {
 		stopWaitFor();
 
-		var window = _root.skillhive.i_SkillhiveBackground.i_Background;
-		var screenWidth:Number = Stage["visibleRect"].width;
-		var backgroundWidth:Number = window._width;
-		window._width = screenWidth;
-		window._x -= (screenWidth - backgroundWidth) / 2;
-	}
-
-	public function revert() : Void {
-		stopWaitFor();
-		
-		var window = _root.skillhive.i_SkillhiveBackground.i_Background;
-		if (window != undefined) {
-			window._width = 2500;
-			window._x = 0;
-		}
+		var window = _root.skillhive;
+		window.m_DefaultSkillhiveBackgroundWidth = window.i_SkillhiveBackground.i_Background._width = Stage["visibleRect"].width;
+		//Would love to know why calling these methods directly does not work, but adding a timeout does....
```

Comment:
It could be an async loading issue.  Just because _root.skillhive.i_SkillhiveBackground.i_Background exists (which is a movieclip on the timeline), doesn't mean the class has been instantiated at that point.  The "code" part of Flash happens in the next frame after the object has been created on the timeline.  That's why if I need to interact with code in another module, I have my WaitFor tests check for the existence of a function in that module, rather than a movieclip.

That's just a guess in this case, as it is something that happens a lot.